mobsf/StaticAnalyzer/views/ios/db_interaction.py

```diff
@@ -38,6 +38,8 @@ def get_context_from_db_entry(db_entry):
             'binary_info': python_dict(db_entry[0].BINARY_INFO),
             'permissions': python_dict(db_entry[0].PERMISSIONS),
             'ats_analysis': python_list(db_entry[0].ATS_ANALYSIS),
```

Comment:
Can we reuse `ats_analysis` for storing `transport_security_info` and `adnetworkidentifiers`. I would prefer not to create a new model field unless it's absolutely necessary.

something like 

```
ats_analysis : { 

  'ats': list(),
  'transportsecurityinfo': list(),
  'adnetworkidentifiers': dict{},
  }
  ```